R/UK.R

```diff
@@ -339,36 +355,31 @@ UK <- R6::R6Class("UK", # rename to country name
         )
         names(self$query_filters) <- self$resolution
       } else {
-        stop(paste("UK data not supported for level", self$level))
+        stop(paste("Uk data not supported for level", self$level))
```

Comment:
I think your merge history has got confused. This is trying to revert changes made on `dev` perhaps merge in again?